### DISCUSSION TITLE
Twitter Yorum ve Beğeni Puanları

### DISCUSSION THREAD

**Discussion by hbozkurtt** [IBAX-io/go-ibax#736] [Soru Cevap - Türkçe]
Arkadaşlar etkinlik açıklamasında retweet yaptığımız gönderileri beğenip 3 arkadaş etiketlediğimizde 200 puan alacağımız söyleniyor bütün gönderilere yaptım fakat hiç puan gelmedi birde 60 gün boyunca her gün 1 şanslı kişiye 50.000 ibxc verileceği söyleniyor bu kişiler kim nerede paylaşılıyor bu sorularımıza cevap verebilecek varsa bilgilendirebilirmi

**Comment by hes91** [comment]
Sen olayı yanlış anlamışsın airdrop gönderileri diye belirtiyo.Onlarda 1 Şubattan önce kendi profiline gelir.

**Reply by hbozkurtt** [reply]
![F14343ED-5A46-4F8F-A91F-1085896621EC](https://user-images.githubusercontent.com/96335500/150986399-c3827184-f822-4509-a1cb-500ca5156a48.jpeg)

**Reply by hbozkurtt** [reply]
Görüntüde işaretledim bu ne anlama geliyor

**Reply by hes91** [reply]
Airdrop gönderileri.Yani retweetledğin gönderiler.

**Comment by hbozkurtt** [comment]
Açıklama beğeni ve 3 arkadaş etiketine ekstra 200 puan diyor fakat öyle bir puan gelmedi bana
